mobile/lib/contracts/user_profile/contracts.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:tree_of_a_kind/features/cqrs/cqrs_action.dart';
+
+part 'contracts.g.dart';
+
+@JsonSerializable()
+class UserProfileDTO {
+  final String id;
+  final String firstName;
+  final String lastName;
+  final DateTime birthDate;
```

Comment:
Idk whether Fluent has JustDate out of the box, but if it does please change it.

Reply:
No it doesn't, nor does c# at the moment.